src/ssl.c

```diff
@@ -42367,6 +42652,16 @@ void* wolfSSL_GetDhAgreeCtx(WOLFSSL* ssl)
     defined(WOLFSSL_HAPROXY)
 
 #ifndef NO_SHA
+    /* One shot SHA hash of message.
+     *
+     * Wrap SHA1 one shot
+     */
+      unsigned char *wolfSSL_SHA(const unsigned char *d, size_t n,
```

Comment:
Make it a define not a function.

Reply:
removed function body and definition, and make  SHA one shot definition to wolfSSL_SHA1